sklearn/tests/test_calibration.py

```diff
@@ -542,3 +542,19 @@ def test_calibration_attributes(clf, cv):
         classes = LabelEncoder().fit(y).classes_
         assert_array_equal(calib_clf.classes_, classes)
         assert calib_clf.n_features_in_ == X.shape[1]
+
+
+# FIXME: remove in 0.26
+def test_calibrated_classifier_cv_deprecation(data):
+    """Check that we raise the proper deprecation warning if accessing
+    `calibrators_` from the `_CalibratedClassifier`."""
```

Comment:
Nit but I feel like we should favor consistency, i.e. keep using comments until we decide to switch entirely

Reply:
I anticipated your proposal :)